paper.tex

```diff
@@ -476,137 +476,156 @@ \section*{Appendix}
-model could be similar to a soft selection model.  Relevant literature to dig up: Felsenstein's review,
-which will lead to other stuff.}
+% \krtcomment{
+%     We need to revisit the paragraph(s) below.
+%     We do not assume individuals are exchangeable, because we do not
+% 	require that all individuals have the same fitness.
+%     See, e.g., p 60 of Wakeley for why fitness == exchangeability
+% 	and why that is required for the standard coalescent.
+% }
+% \mhcomment{The `exchangeability' assumption also breaks down under 
+% 	different reproductive modes, since the sampling of the alleles
+%     affects coalescent times. 
+% 	For example, in a highly selfing species the coalescent time for two different alleles 
+% 	within an individual is much lower than two alleles from different individuals.
+%     I've triedto rewrite the below paragraph to update and clarify
+%     the reproduction parameters in line with the discussions on Github.
+%     Finally, 'rates' imply events occur as Poisson-like 
+% 	processes over continuous time, whereas the values used here instead define the 
+% 	\emph{fraction} of reproductions of each type.
+%     Shall we rename to selfing/clonal `fractions',
+% 	or would that mess up the rest of the schema too much?
+% }
+
+Each population has an assigned selfing rate and cloning rate, where each defines
+the probability that offspring are generated from one generation to the next by either
+self-fertilisation or cloning of an individual.
+More specifically, for a given epoch within a population denote the clonal rate by
+$\sigma$ and the selfing rate by $S$.
+$S$ and $\sigma$ can take any value between zero and one and
+can sum to more than one.
+Each generation a proportion of offspring
+$\sigma$ are expected to be generated through clonal reproduction,
+while $1-\sigma$ are expected to arise through sexual reproduction.
+Within the sexually-reproduced offspring,
+$S$ are born via self-fertilisation while the rest
+have parents drawn at random from the previous generation.
+Depending on the simulation software, this random drawing of parent may occur
+either with or without replacement. When drawing occurs with replacement, a small
+amount of ``residual'' selfing is expected, so that the realized selfing probability
+is $(1-\sigma)(S + (1-S)/N)$ instead of $(1-\sigma)S$ (so that even with $\sigma=0$
+and $S=0$, selfing may still occur with probability $1/N$).
+Simulation software that allow variable rates of selfing are expected to clearly
+document the expected behaviour.
+
+By allowing the definition of selfing and cloning probabilities, we allow many 
+standard models to be defined.
+However, by parameterising selfing and cloning as we have, we assume that these
+properties of populations can be specified independently from the genetics.
+In other words, mutations that cause selfing probabilities to fluctuate within
+an epoch are not considered.
+\aprcomment{
```

Comment:
I think we want a section in the Discussion outlining some obvious things that people can do w/demes that are sitting "just outside" the strict specification: hard selection, etc..

Reply:
I'll add a comment to that point here.